airflow/www/templates/airflow/task_instance.html

```diff
@@ -60,13 +61,15 @@ <h4>
 {% endblock %}
 {% block tail %}
   {{ super() }}
+  <script src="{{ url_for_asset('utils.js') }}"></script>
```

Comment:
Dup include?

Reply:
We have already discussed this in the past. Double imports in rendered file mean that this views needs libraries several times in several views. This will be more readable when the code will be in separate files and  we could use a ["import" statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import). Currently, we must use such syntax.
https://github.com/apache/airflow/pull/4787#discussion_r267573261

Reply:
We have two identical script tags - once here and again in the top level template. Even ignoring import statement we don't need a duplicate script tag do we?